### ISSUE TITLE
Both validation (part of training) and test are done with val data?

### ISSUE THREAD

**Issue by dusa2** [open-mmlab/mmskeleton#276]
In the config file, for kinetics skeleton, the training uses batches from the validation dataset during eval steps, and the test step uses the validation dataset in its entire.

Shouldn't the validation set be different from the final test set? Isn't the training using validation steps to optimize training using the val_loss? Or is the training independent of this val_loss? or any kind of regularization? fine-tune the model hyperparameters? and therefore, the validation dataset?

**Comment by antoniolq** [comment]
I have the same question，Can  you tell how to understand it？
